dbms/src/TableFunctions/TableFunctionRandom.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <TableFunctions/ITableFunction.h>
+
+namespace DB
+{
+/* random(structure, limit) - creates a temporary storage filling columns with random data
+ * random is case-insensitive table function
+ */
+class TableFunctionRandom : public ITableFunction
```

Comment:
Let's rename the class and adjust the comments.

Reply:
done